test/binaryen.js/expressions.js

```diff
@@ -1207,6 +1207,68 @@ console.log("# SIMDLoad");
   module.dispose();
 })();
 
+console.log("# SIMDLoadStoreLane");
+(function testSIMDLoadStoreLane() {
+  const module = new binaryen.Module();
+
+  var op = binaryen.Operations.Load8LaneVec128;
+  var offset = 16;
+  var index = 1;
+  var align = 2;
+  var ptr = module.i32.const(1);
+  var vec = module.v128.const([1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16]);
+  const theSIMDLoadStoreLane = binaryen.SIMDLoadStoreLane(module.v128.load8_lane(offset, align, index, ptr, vec));
+  assert(theSIMDLoadStoreLane instanceof binaryen.SIMDLoadStoreLane);
+  assert(theSIMDLoadStoreLane instanceof binaryen.Expression);
+  assert(theSIMDLoadStoreLane.op === op);
+  assert(theSIMDLoadStoreLane.offset === offset);
+  assert(theSIMDLoadStoreLane.align === align);
+  assert(theSIMDLoadStoreLane.index === index);
+  assert(theSIMDLoadStoreLane.ptr === ptr);
+  assert(theSIMDLoadStoreLane.vec === vec);
+  assert(theSIMDLoadStoreLane.type === binaryen.v128);
+  assert(theSIMDLoadStoreLane.store === false);
+
+  theSIMDLoadStoreLane.op = op = binaryen.Operations.Load16LaneVec128;
+  assert(theSIMDLoadStoreLane.op === op);
+  theSIMDLoadStoreLane.offset = offset = 32;
+  assert(theSIMDLoadStoreLane.offset === offset);
+  theSIMDLoadStoreLane.align = align = 4;
+  assert(theSIMDLoadStoreLane.align === align);
+  theSIMDLoadStoreLane.index = index = 2;
+  assert(theSIMDLoadStoreLane.index === index);
+  theSIMDLoadStoreLane.ptr = ptr = module.i32.const(2);
```

Comment:
How about using a different number for the index and the pointer arguments to make sure the test shows that they end up in the right places?

Reply:
It kinda does that, by initializing with `index = 1` and `ptr = i32.const(1)`, testing that, and then updating to `index = 2` and `ptr = i32.const(2)` before testing again here. I may be missing something though :)

Reply:
Ah yes, fair enough. I was only thinking about the values that get printed.